RadixWallet/WalletApp.swift

```diff
@@ -6,15 +6,6 @@ import SwiftUI
 struct WalletApp: SwiftUI.App {
 	@UIApplicationDelegateAdaptor var delegate: AppDelegate
 
-	init() {
-		#if DEBUG
-		// MUST NOT be called twice (panics...)
-		// MUST NOT be enabled in PROD (not ensured that secrets are omitted)
-		// This is VERY VERY verbose, so mosly useful for debugging.
-		enableLoggingFromRust()
```

Comment:
We should use SargonOS loggin systems instead. So I removed that freestanding enable log function in Sargon.